.github/workflows/ci.yml

```diff
@@ -48,48 +64,55 @@ jobs:
         - windows: py10
         - macos: py311
         - linux: py310-oldestdeps
+    secrets:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
   docs:
     needs: [core]
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
     with:
+      default_python: '3.12'
       submodules: false
       pytest: false
-      default_python: '3.12'
       toxdeps: "tox-pypi-filter"
       libraries: |
         apt:
           - graphviz
       envs: |
         - linux: build_docs
 
-  online:
```

Comment:
I suspect we need this online job and not the cron in this repo.

Reply:
yeah online stays as I need to test jsoc and the sunpy tests.